### ISSUE TITLE
Allow to specify paths that sould be excluded/included when performing scan :racehorse: 

### ISSUE THREAD

**Issue by jamacku** [redhat-plumbers-in-action/differential-shellcheck#231]
### Type of issue

Feature Request

### Description

This feature would allow users to run ShellCheck only on a subset of shell scripts, for example, exclude `tests` from the scan. Typically testing frameworks like `bats` or `beakerlib` can produce many false positives.

### Describe the solution you'd like

It would be nice to have something like this:

```yml
with:
  - include: src/**
  - exclude: test/**
```
